src/cli.ts

```diff
@@ -6,36 +6,38 @@ import { DiagnosticSeverity } from 'vscode-languageserver';
     .option('files', { type: 'array', description: 'The list of files (or globs) to include in your project. Be sure to wrap these in double quotes when using globs.' })
     .option('host', { type: 'string', description: 'The host used when deploying to a Roku.' })
     .option('ignore-error-codes', { type: 'array', description: 'A list of error codes that the compiler should NOT emit, even if encountered.' })
-    .option('log-level', { type: 'string', default: 'log', description: 'The log level. Value can be "error", "warn", "log", "info", "debug".' })
+    .option('log-level', { type: 'string', defaultDescription: '"log"', description: 'The log level. Value can be "error", "warn", "log", "info", "debug".' })
     .option('out-file', { type: 'string', description: 'Path to the zip folder containing the bundled project. Defaults to `./out/[YOUR_ROOT_FOLDER_NAME].zip' })
     .option('password', { type: 'string', description: 'The password for deploying to a Roku.' })
     .option('project', { type: 'string', description: 'Path to a bsconfig.json project file.' })
-    .option('retain-staging-folder', { type: 'string', default: false, description: 'Prevent the staging folder from being deleted after creating the package.' })
+    .option('retain-staging-folder', { type: 'boolean', defaultDescription: 'false', description: 'Prevent the staging folder from being deleted after creating the package.' })
     .option('root-dir', { type: 'string', description: 'Path to the root of your project files (where the manifest lives). Defaults to current directory.' })
     .option('staging-folder-path', { type: 'string', description: 'The path where the files should be staged (right before being zipped up).' })
-    .option('username', { type: 'string', default: 'rokudev', description: 'The username for deploying to a Roku.' })
+    .option('username', { type: 'string', defaultDescription: '"rokudev"', description: 'The username for deploying to a Roku.' })
     .option('source-root', { type: 'string', description: 'Override the root directory path where debugger should locate the source files. The location will be embedded in the source map to help debuggers locate the original source files. This only applies to files found within rootDir. This is useful when you want to preprocess files before passing them to BrighterScript, and want a debugger to open the original files.' })
-    .option('watch', { type: 'boolean', default: false, description: 'Watch input files.' })
+    .option('watch', { type: 'boolean', defaultDescription: 'false', description: 'Watch input files.' })
     .strict()
     .check(argv => {
-        let d = argv.diagnosticLevel;
-        if (d !== 'error' && d !== 'warn' && d !== 'hint' && d !== 'info') {
-            throw new Error(`Invalid diagnostic level "${d}". Value can be "error", "warn", "hint", "info".`);
+        const diagnosticLevel = argv.diagnosticLevel as string;
```

Comment:
Happy to see `const` here - I thought I'd stay on the all `let` you had ;)

Reply:
Yeah...you convinced me. I've started using const other places too. Eventually I'll get around to converting most of the `let`s to `const`.